build.gradle

```diff
@@ -95,6 +95,8 @@ subprojects {
 
     dependencies {
         compile "org.slf4j:slf4j-api:$slf4jVersion"
+        compile "org.apache.zookeeper:zookeeper:$zookeeperVersion"
```

Comment:
QQ: I think we interact with zookeeper only via helix library client. Would we need this direct dependency?

Reply:
Either way works but I prefer this as it is upfront and explicit. And if we import zookeeper directly, we are guaranteed we use the latest version.

Reply:
Actually I prefer always leveraging the transitive dependency if it works since we don't need to take care of upgrading this version all the time. Let's say when we upgrade the helix lib next time, we don't need to remember to upgrade this zk version.

Reply:
removed